src/core/core_utils.js

```diff
@@ -548,6 +549,27 @@ function numberToString(value) {
   return value.toFixed(2);
 }
 
+function getNewlyAddedAnnotations(annotationStorage) {
```

Comment:
I'd prefer the following instead, since it tells you what sort of data is returned (and it's slightly shorter):
```suggestion
function getNewAnnotationsMap(annotationStorage) {
```